install/swiftly-install.sh

```diff
@@ -48,6 +51,47 @@ read_input_with_default () {
     fi
 }
 
+yn_prompt () {
```

Comment:
I originally thought we needed another y/n prompt, so factored it out into these functions. We didn't end up needing it, but it still seemed useful so I left it in.